app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/repository/RepositoryBrowserViewModel.kt

```diff
@@ -18,8 +18,8 @@ class RepositoryBrowserViewModel : BaseViewModel() {
     private val repo: BitbucketRepository by inject()
 
     // State
-    private val srcFiles = MutableStateFlow<List<RepoFile>>(emptyList())
-    private var currentRepoName: String = ""
+    val srcFiles = MutableStateFlow<List<RepoFile>>(emptyList())
+    var currentRepoName: String = ""
```

Comment:
In general, we don't want to make items public just for testing purpose.  Try to get same coverage by testing what these elements depend on and keeping these private.

Reply:
Will do!

Reply:
@MMurrayAtBR I agree with @BottleRocket-Colin's original statement here that it would be best if these properties were left as private. I see that you've added the `@VisibleForTesting` annotation instead, which isn't ideal but should be okay if we use it sparsely.

Just keep in mind for future tests that it is always preferred to prevent making visibility changes just to make testing easier over changing the visibility and using the annotation :)